src/features/LoginModal/LoginForm/LoginForm.tsx

```diff
@@ -34,25 +34,48 @@ const LoginForm: React.FC<LoginFormProps> = (props) => {
             const user = await signInWithEmailAndPassword(auth, email, password)
             loginUser.mutate({ id: user.user.uid, email: user.user.email, displayName: user.user.displayName }, {
                 onError: (err: any) => {
+                    setIsLoading(false)
```

Comment:
Should set error here so its displayed, but believe there needs to be one sent from the backend first